### DISCUSSION TITLE
S3 mount from labels not working

### DISCUSSION THREAD

**Discussion by fcollonval** [datashim-io/datashim#348] [Q&A] [answered]
Hey,

I'm testing this very promising feature (thanks for the hard work). But I'm unable to figure out what is going wrong.

Following the installation instructions, I can create successfully a _Dataset_ - the expected pvc, pv, configmap and secret are created. But when I create the nginx pod, the data are not mounted - it works if I mount the pvc explicitly.

I'm still learning k8s - so I'm probably missing some ways to find useful logs. What I see is that there are not _volumeAttachment_ created (seem it should !?).

For reference I'm using kubernetes 1.28.

I tried also with microk8s and the same behavior is happening (same version of kubernetes).

Could someone point me what I should look for to debug that issue?

**Comment by srikumar003** [comment, marked as answer]
@fcollonval You might have not labeled the namespace in which you are creating the pods. You can do this like so:
```
kubectl label ns <namespace> monitor-pods-datasets=enabled
```
where `<namespace>` is the one in which you are creating the pods. Please try this and let us know!

**Reply by fcollonval** [reply]
Hey @srikumar003 sorry for the late answer but indeed this was the missing part.

Thanks again for this awesome piece of code.
